### ISSUE TITLE
No HTTPS connection to blog.

### ISSUE THREAD

**Issue by filfreire** [yegor256/blog#213]
Seems HTTPS connection is not available at the moment:

![screen shot 2017-12-28 at 09 31 29](https://user-images.githubusercontent.com/11976836/34406623-04d0ea7c-ebb2-11e7-805d-8b15a435068c.png)

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by filfreire** [comment]
@yegor256 fyi

**Comment by yegor256** [comment]
@filfreire yeah, I know. Can't really find the right way to implement it.

**Comment by filfreire** [comment]
@yegor256, we could try to create a webservice that commits a new certificate from letsencrypt?
https://letsencrypt.org/getting-started/

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/blog#213` is not assigned, can't get start time

**Comment by 0crat** [comment]
Oops! Job `gh:yegor256/blog#213` was not in scope

**Comment by yegor256** [comment]
@filfreire yes, I'm thinking about something like that. There is cloudflare, which does something like that, but it's rather complex to configure (maybe I'm wrong).

**Comment by rozhok** [comment]
It's easy to configure even on CloudFlare.

Also, consider using Netlify for hosting, it provides certs for free (as well as ton of other features).

**Comment by filfreire** [comment]
@yegor256 I believe this issue is now possible to solve, see
> Custom domains on GitHub Pages gain support for HTTPS
https://blog.github.com/2018-05-01-github-pages-custom-domains-https/

**Comment by 0crat** [comment]
Job `gh:yegor256/blog#213` is not assigned, can't get performer
